src/components/shared/forms/styles/pill.scss

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/themes';
+@import 'src/styles/mixins';
+
+$pillHeight: 30px;
+$pillMinWidth: 127px;
+$pillIconSize: 14px;
+.pill {
+  display: flex;
+  flex-shrink: 0;
+  justify-content: space-between;
+  border: 0;
+  align-items: center;
+  text-align: left;
+  float: right;
+  white-space: nowrap;
+  margin: 7px 7px 0px 0px;
+  cursor: pointer;
+  font-family: inherit;
+  @include rem-fallback(padding, 0px, $pillHeight/4);
+  @include rem-fallback(min-width, $pillMinWidth);
+  p{
+    padding: 0px;
+    margin: 0px;
+  }
+  div{
+    float:right;
```

Comment:
See comment above regarding the button styling and flexbox. General rule, never use float.

Reply:
Removed.